database/migrations/2016_08_02_072458_mct_usert_table_created.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MctUsertTableCreated extends Migration {
+
+	/**
+	 * Run the migrations.
```

Comment:
@Shubham2801 
Can you remove all these migrations for the time being. Just delete them and commit again.
There are some changes being made to the schema currently.
We'll come to the migrants a little while later when all our DB changes has been finalized.